lib/vanagon/cli/sign.rb

```diff
@@ -28,7 +28,7 @@ def run(_)
 
         require 'packaging'
         Pkg::Util::RakeUtils.load_packaging_tasks
```

Comment:
This is likely no longer needed. Since we're not using tasks, we shouldn't need to load them.